lib/autoupdate/tools/updater.go

```diff
@@ -389,11 +389,6 @@ func (u *Updater) downloadHash(ctx context.Context, url string) ([]byte, error)
 // downloadArchive downloads the archive package by `url` and writes content to the writer interface,
 // return calculated sha256 hash sum of the content.
 func (u *Updater) downloadArchive(ctx context.Context, url string, f io.Writer) ([]byte, error) {
-	// Display a progress bar before initiating the update request to inform the user that
-	// an update is in progress, allowing them the option to cancel before actual response
-	// which might be delayed with slow internet connection or complete isolation to CDN.
-	pw, finish := newProgressWriter(10)
-	defer finish()
```

Comment:
Can you explain me why the progress was breaking with aliases? And how showing the progress bar after doing the HTTP call fixes it?

Reply:
I gonna move this one in separate PR, this one related to this [discussion](https://gravitational.slack.com/archives/C01TYKHFVTQ/p1733933975926119?thread_ts=1733919010.833779&cid=C01TYKHFVTQ). For macosx we download two tarballs for tctl and tsh, after version v17 we have only one this one cause to show empty progress bar (since we still make request to check it)